### PR TITLE
Replaces the vector class with the one from framework

### PR DESCRIPTION
**Pull request by karlmerten-prodigy**
Replaces the internal vector with the framework one to have less vector classes.
Also gitignores the distribution files and sets up Jfrog properly

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@prodigy/navmesh",
-  "version": "1.0.1",
+  "version": "1.1.0-alpha-083c53c",
```
Comment (ProdigyChrisLight) [inline package.json]: Is this going to be the final version number?

File: package.json [hunk 2]
```diff
@@ -9,10 +9,13 @@
     "test": "echo \"Warning: no test specified\" && exit 0"
   },
   "dependencies": {
+    "@prodigy/game-framework": "7.0.0",
```
Comment (ProdigyChrisLight) [inline package.json]: Is this dependency going to get built into the web worker code and bloat it? Or will it all get tree-shaken out except for Vector2?

Comment (ProdigyChrisLight) [reply package.json]: We could just use a type compatible with framework's Vector2 to avoid this.

**Review by ProdigyChrisLight — COMMENTED** [review]
Looks good. A few questions.

**Review by ProdigyChrisLight — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (karlmerten-prodigy) [reply package.json]: Nope, just JFrog things

**Pull request merged**